webpack.config.js

```diff
@@ -74,7 +74,7 @@ const generateConfig = ({extensionPath, devMode=false, customOutputPath, analyze
   });
   /* gzip everything - https://github.com/webpack-contrib/compression-webpack-plugin */
   const pluginCompress = new CompressionPlugin({
-    filename: "[path].gz[query]",
+    filename: "[path][base].gz",
     algorithm: "gzip",
```

Comment:
Something weird is going on with code splitting not happening on `optimization.splitChunks.cacheGroups.coreVendors`.

The output follows `filename` instead of `chunkFilename` here:

https://github.com/nextstrain/auspice/blob/1d33b1e667320ba2c272054ba28a330f4c2d2c83/webpack.config.js#L189-L190

So the file can't be found by bundlesize ([link](https://github.com/nextstrain/auspice/runs/8260498558?check_suite_focus=true#step:5:5)):

```
 ERROR  There is no matching file for ./dist/auspice.chunk.core-vendors.bundle.*.js in /home/runner/work/auspice/auspice 
```

I tried adjusting `minChunks` and `minSize` but still can't get code splitting to happen. Putting this investigation off for another day.

Reply:
Seeme relevant: https://github.com/webpack/webpack/issues/9297

Reply:
Still don't fully understand this. Using a workaround: ae8ce43bc818a3dfb304eb55092c175b7ad4955d

Reply:
It's been a long time since I delved into the depths of webpack (and that was v4) so I don't understand what's happening here either!

The idea behind the coreVendors chunk is that it has a longer cache timeout, and thus browsers don't need to fetch it as often. This presumes that it's more stable between auspice releases than other chunks (which bundle up our code). 

Given our cadence of auspice releases and the actual sizes involved (the largest chunk is ~200kb uncompressed) I don't know if this development complexity is warranted anymore, but you seem to have got it working correctly again so we should be good for another few years!!!

Reply:
It's been a long time since I delved into the depths of webpack (and that was v4) so I don't understand what's happening here either!

The idea behind the coreVendors chunk is that it has a longer cache timeout, and thus browsers don't need to fetch it as often. This presumes that it's more stable between auspice releases than other chunks (which bundle up our code).

Given our cadence of auspice releases and the actual sizes involved (the largest chunk is ~200kb uncompressed) I don't know if this development complexity is warranted anymore, but you seem to have got it working correctly again so we should be good for another few years!!!